cmd/notation/verify.go

```diff
@@ -105,7 +104,7 @@ func verifySignatures(ctx context.Context, verifier notation.Verifier, manifestD
 		}
```

Comment:
If reading one path failed, should we continue to read signatures in other paths instead of failing the verification right away?

Reply:
Continue reading would be better, updated.